From: NAStools <dev@nas.tools>
Date: Wed, 20 Apr 2016 15:59:47 -0700
Subject: Add some ReadyNAS specific process groups

---
 conf.d/apps_groups.conf | 26 ++++++++++++++++++++++----
 1 file changed, 22 insertions(+), 4 deletions(-)

diff --git a/conf.d/apps_groups.conf b/conf.d/apps_groups.conf
index 995ee5d..6381b09 100644
--- a/conf.d/apps_groups.conf
+++ b/conf.d/apps_groups.conf
@@ -62,7 +62,7 @@ splunk: splunkd
 mongo: mongod
 lighttpd: lighttpd
 ftpd: proftpd in.tftpd
-samba: smbd nmbd winbindd
+samba: smbd nmbd winbindd smbd-notifyd
 nfs: rpcbind rpc.* nfs*
 ssh: ssh* scp
 X: X lightdm xdm pulseaudio gkrellm
@@ -75,7 +75,7 @@ ntp: ntp*
 deluge: deluge*
 vbox: vbox* VBox*
 log: ulogd syslog* rsyslog* logrotate
-nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid
+nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid readynas-agent
 ppp: ppp* pptp*
 inetd: inetd xinetd
 openvpn: openvpn
@@ -90,7 +90,25 @@ wifi: hostapd wpa_supplicant
 system: systemd* udisks* udevd connmand ipv6_addrconf dbus-*
 ksmd: ksmd
 lxc: lxc*
-zfs-spl: spl_* 
+zfs-spl: spl_*
 zfs-posix: z_*
 zfs-txg: txg_* zil_*
-zfs-arc: arc_* l2arc* 
+zfs-arc: arc_* l2arc*
+readynasd: dbbroker.cgi fsbroker.cgi readynasd
+btrfs: btrfs*
+mdcsrepair: mdcsrepaird
+fvbackup: fvbackup*
+afp: netatalk afpd cnid_metad
+ReadyDLNA: minidlnad
+avahi: avahi-daemon
+crypto: crypto
+raid: md* raid*
+raidar: raidard
+networking: leafp2p connmand
+upnp: upnphttpd
+cloud: remote rdhook.sh rdbrokerd readydropd rcbrokerd
+fvdropbox: fvdropbox
+fvamazon: fvamazon
+antivirus: ctscand avdu
+itunes: forked-daapd
+spindown: noflushd
