From: NAStools <dev@nas.tools>
Date: Wed, 2 Nov 2016 17:58:37 -0700
Subject: Add some ReadyNAS specific process groups

---
 conf.d/apps_groups.conf | 26 ++++++++++++++++++++++++--
 1 file changed, 24 insertions(+), 2 deletions(-)

diff --git a/conf.d/apps_groups.conf b/conf.d/apps_groups.conf
index 57357a8..5e85bb5 100644
--- a/conf.d/apps_groups.conf
+++ b/conf.d/apps_groups.conf
@@ -120,13 +120,13 @@ xmpp: *vines* *prosody*
 # monitoring
 
 logs: ulogd* syslog* rsyslog* logrotate
-nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid watchdog tailon nrpe
+nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid watchdog tailon nrpe readynas-agent
 splunk: splunkd
 
 # -----------------------------------------------------------------------------
 # file systems and file servers
 
-samba: smbd nmbd winbindd
+samba: smbd nmbd winbindd smbd-notifyd
 nfs: rpcbind rpc.* nfs*
 zfs: spl_* z_* txg_* zil_* arc_* l2arc*
 btrfs: btrfs*
@@ -217,3 +217,25 @@ sidekiq: *sidekiq*
 java: java
 chat: irssi
 ipfs: ipfs
+
+# -----------------------------------------------------------------------------
+# ReadyNAS
+
+readynasd: dbbroker.cgi fsbroker.cgi readynasd
+btrfs: btrfs*
+mdcsrepair: mdcsrepaird
+fvbackup: fvbackup*
+afp: netatalk afpd cnid_metad
+ReadyDLNA: minidlnad
+avahi: avahi-daemon
+crypto: crypto
+raid: md* raid*
+raidar: raidard
+networking: leafp2p connmand
+upnp: upnphttpd
+cloud: remote rdhook.sh rdbrokerd readydropd rcbrokerd
+fvdropbox: fvdropbox
+fvamazon: fvamazon
+antivirus: ctscand avdu
+itunes: forked-daapd
+spindown: noflushd
